ReactNativeTraining/src/app/screens/components/ContactList/index.js

```diff
@@ -12,6 +12,7 @@ export default class ContactList extends Component {
       hideChevron
       roundAvatar
       avatar={item.avatar}
+      onPress={() => this.props.onClick(item)}
```

Comment:
declare `handlePress` above and use here, `onPress={this.handlePress}` to prevent re-renders